core/lib/spree/core/state_machines.rb

```diff
@@ -70,7 +70,7 @@ def order
         @order ||= begin
           if Spree::Config.use_legacy_order_state_machine
             Spree::Deprecation.warn(
-              "Spree::Order state machine defined in Spree::Order::Checkout is deprecated." \
+              "Spree::Order state machine defined in Spree::Order::Checkout is deprecated.\s" \
```

Comment:
I think we usually go with a regular space rather than the escape character.

Reply:
Sounds good. 😁 I'll push that up after I grab coffee this morning.